Library/Homebrew/formula.rb

```diff
@@ -1766,6 +1766,7 @@ def to_hash
     hsh = {
       "name"                     => name,
       "full_name"                => full_name,
+      "tap"                      => tap.name,
```

Comment:
```suggestion
      "tap"                      => tap&.name,
```

Same here: `tap` can be `nil` so we need to check for that.

Reply:
Perfect! I'll make the change!

Reply:
Done!